electronID preliminary in 94X

cgalloni opened the pull request:
Dear @gitytakahas , @IzaakWN , @zucchett ,@dpinna,

I updated the Electron ID floowing the preliminary recommendations:
https://hypernews.cern.ch/HyperNews/CMS/get/egamma/2013.html

-@gitytakahas , @IzaakWN I inserted the MVA ID without the isolation. Is it ok?
-@zucchett ,@dpinna, what about the GSF electron instead of the pat?

cgalloni committed:
electronID preliminary in 94X

cgalloni committed:
adding samples v2

dpinna commented:
@cgalloni : I have checked the part of the electrons where the dEtaInSeed is calculated, eg:
https://github.com/cgalloni/EXOVVNtuplizerRunII/blob/0d192e91dd935e70e8de820665a192a54707199b/Ntuplizer/plugins/ElectronsNtuplizer.cc#L385

In case we want to move to the Gsf electrons for this var calculation we would need another definition for ele. Ele as PAT electron is used in the rest of the code for many things, so it might be a good idea to leave also the dEtaInSeed calculation here with PAT electrons. The difference is small wrt the calculation using Gsf electron and such variable will be used only if we need to change the selection value for the ID.

cgalloni committed:
preliminary 94X sample production

cgalloni committed:
fixing ak8 jet pt 170 threshold

cgalloni committed:
adding deepCSV and JEC MC Fall17

cgalloni commented:
Hi All, 
it seems a huge PR because nobody approved it since quite some time. 
The latest things added are:
-corrected bool-> float for 1 tau ID raw output
-added deepCSV : deep_csv_b and deep_csv_bb need to be added while checking the b-tag WP
-added preliminary JEC for Fall17 MC

Please review it. Otherwise I'll merge it. 
@gitytakahas , @dpinna, @zucchett, @IzaakWN @cgalloni
